src/goto-harness/recursive_initialization.cpp

```diff
@@ -264,11 +275,12 @@ irep_idt recursive_initializationt::build_constructor(const exprt &expr)
     size_symbol_expr,
     // the expression name may be needed to decide if expr should be treated as
     // a string
-    expr_name);
+    expr_name,
+    is_nullable);
 
   goto_model.goto_functions.function_map[function_symbol.name].type =
     to_code_type(function_symbol.type);
-  return type_constructor_names[type];
+  return type_constructor_names.at(key);
```

Comment:
Is there a reason why this access is checked but the above are not?

Reply:
Not really, tbh we'd considered refactoring this entire section to avoid multiple lookup of the same key. We only added the `.at` because at some point we were hitting a case where keys were missing from the map.